github.js

```diff
@@ -16,6 +17,8 @@ var semver = require('semver');
 
 var which = require('which');
 
+var merge = require('merge')
```

Comment:
I hate to be difficult but I try to avoid dependencies where possible.

Perhaps just - 

```javascript
function extend(a, b) {
  for (var p in b)
    a[p] = b[p];
}
```

?

I know it doesn't cover every case, but it will cover the needs here?